tests/__init__.py

```diff
@@ -60,11 +60,22 @@ def get_qualified_user_id(*args):
     if config:
         config_parsed = TokenAuthenticator.parse_config(config)
     else:
-        config_parsed = TokenAuthenticator.parse_config({"secret": "foxies"})
+        config_parsed = TokenAuthenticator.parse_config(
+            {
+                "jwt": {"secret": "foxies"},
+                "oidc": {
+                    "issuer": "https://idp.example.org",
```

Comment:
example.org is a valid domain and something is hosted there, can we use `.test` or something? :D